### ISSUE TITLE
Invalide password starting with real password is considered valid

### ISSUE THREAD

**Issue by ThomasBaruzier** [novnc/noVNC#1758]
**Describe the bug**
If my password is password, password123 will let me in
...

**To Reproduce**
Try to log in with the right password with extra characters

**Expected behavior**
Consider it a wrong password

**Client (please complete the following information):**
 - OS: Latest Arch linux 
 - Browser: Any
 - Browser version: Tested on latest firefox, safari, chromium, chrome

**Server (please complete the following information):**
 - noVNC version: [e.g. 1.0.0 or git commit id]
 - VNC server: TigerVNC
 - WebSocket proxy: websockify + nginx

**Comment by ThomasBaruzier** [comment]
Sorry for the typo in the title (French habits), i can't change it

**Comment by CendioOssman** [comment]
Indeed. However, we could probably be better about warning users that aren't that familiar with VNC.

**Comment by CendioOssman** [comment]
Also see TigerVNC/tigervnc#370 for the same discussion in that project.
